tests/specs/lockfile/frozen_lockfile/sub-dynamic.ts

```diff
@@ -0,0 +1 @@
+const { subtract } = await import("npm:" + "@denotest/subtract@1");
```

Comment:
Instead of doing string concatenation, maybe assign one of these to a variable then do concatenation. I think there was some talk about making this analyzable in the future 😅